staging/src/k8s.io/apiserver/pkg/util/flowcontrol/apf_controller.go

```diff
@@ -359,10 +425,20 @@ func (cfgCtl *configController) lockAndDigestConfigObjects(newPLs []*fctypesv1a1
 
 	meal.finishQueueSetReconfigsLocked()
 
+	if cfgCtl.embargoTime != nil && len(meal.fsStatusUpdates) == 0 && !meal.suppressedWrites {
```

Comment:
why does whether or not writes were suppressed impact the clearing of the embargo?

Reply:
Added a comment explaining what's going on here.